x-pack/plugins/cases/server/services/user_actions/operations/create.ts

```diff
@@ -375,21 +376,16 @@ export class UserActionPersister {
   }
 
   public async createUserAction<T extends keyof BuilderParameters>({
-    action,
-    type,
-    caseId,
-    user,
-    owner,
-    payload,
-    connectorId,
-    attachmentId,
+    userAction,
```

Comment:
Small refactor to separate the data from the options.

Reply:
This was a bit confusing in the PR 😅 Does it actually affect the changes you will need for the subactions?